vignettes/tech-dm-keyed.Rmd

```diff
@@ -0,0 +1,216 @@
+---
+title: "Manipulating individual tables"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Technical: Manipulating individual tables}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+``````{r setup, include = FALSE}
+source("setup/setup.R")
+``````
+
+This vignette deals with situations where you want to transform tables of your `dm` and then update an existing table or add a new table to the `dm`.
+There are two straightforward approaches:
```

Comment:
```suggestion
There are two approaches:
```

I feel this adjective sounds like "you just need to" which is not beginner friendly.